### DISCUSSION TITLE
Gitlab self-managed: send preview URL to Gitlab UI somehow

### DISCUSSION THREAD

**Discussion by thomasmattheussen** [vercel/vercel#8658] [Help] [answered]
When using Vercel in Gitlab self-managed with this guide: https://vercel.com/guides/how-can-i-use-gitlab-pipelines-with-vercel

Is there some way to get the preview URL on which your site is deployed inside the gitlab UI? 
Maybe in a comment on the merge request?
Or instead of going to the job log by clicking this button, you'd go to the URL? 
<img width="251" alt="image" src="https://user-images.githubusercontent.com/1071255/193262635-1394879c-909c-4c28-bede-8a2925049f0e.png">

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @thomasmattheussen! Sorry for the delayed response. I'm not sure of a way to get the preview URL in the GitLab UI. The [pipeline docs](https://docs.gitlab.com/ee/ci/pipelines/) might offer a way to update the UI with the deployment URL. And folks over in the [GitLab forum](https://forum.gitlab.com/) know much more about pipeline cababilities, so it would be worth asking there to see if they have a solution for you!

**Reply by thomasmattheussen** [reply]
Ok, thanks anyway!
